src/templates/base.html

```diff
@@ -223,6 +242,22 @@ <h4 class="ui inverted header">Footer Header</h4>
         }
         // Django returns <input type="hidden" value="csrf-token-secret"> so we just grab the `value` attribute from this
         CSRF_TOKEN = $('{% csrf_token %}').attr('value')
+
+        $(document).ready( function () {
+            $('.ui.search').search({
+                apiSettings: {
+                    onResponse: function(codalabResponse) {
+                        _.forEach(codalabResponse, (response) => {
+                            response.url = '/competitions/' + response.id
```

Comment:
Should be
`response.url = URLS.COMPETITION_DETAIL(response.id)` so that this isn't hardcoded.